channels-rke2.yaml

```diff
@@ -572,3 +614,29 @@ releases:
       rke2-ingress-nginx:
         repo: rancher-rke2-charts
         version: 4.1.002
+  - version: v1.23.7-rc1+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-23-7-rc1-rke2r1
+      <<: *serverArgs-v1-23-6-rke2r2
+      egress-selector-mode:
```

Comment:
@rancher-max Do you know if this new server arg is supposed to be in KDM?

Reply:
As a new server arg in this version, it should be yes. Thank you!